backend/core/src/listings/listings.controller.ts

```diff
@@ -57,6 +57,20 @@ export class ListingsController {
     return mapTo(PaginatedListingDto, await this.listingsService.list(queryParams))
   }
 
+  @Get("includeExternal")
+  @ApiExtraModels(ListingFilterParams)
+  @ApiOperation({
+    summary: "List listings and optionally include external listings",
+    operationId: "listIncludeExternal",
+  })
+  @UseInterceptors(ClassSerializerInterceptor)
+  @UsePipes(new ValidationPipe(defaultValidationPipeOptions))
+  public async getAllWithExternal(
+    @Query() queryParams: ListingsQueryParams
+  ): Promise<PaginatedListingDto[]> {
```

Comment:
(probably copypasta from the other branch) I'm not sure this is the right type, does it need to be an array?